scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -48,9 +49,13 @@ function prepare_vm {
 
     popd
 
-    wget https://aka.ms/downloadazcopy-v10-linux
-    tar xvf downloadazcopy-v10-linux
-    ./azcopy_linux_amd64_10.1.2/azcopy cp "${kernel_version}" "https://${storage_account}.blob.core.windows.net/kernel/${storage_sas_token}" --recursive
+    storage_account=$(echo $storage_account|sed -e 's/[\r\n]//g')
```

Comment:
why you need to sanitize the variables inside the script?

Reply:
when reading variables from secret file \r and  \n are appended to the values, so used sed to remove these characters